pipenv/core.py

```diff
@@ -75,20 +74,22 @@
         INSTALL_LABEL = "🎅   "
     else:
         INSTALL_LABEL = "🐍   "
-    INSTALL_LABEL2 = crayons.normal("☤  ", bold=True)
+    INSTALL_LABEL2 = click.style("☤  ", bold=True)
     STARTING_LABEL = "    "
 else:
     INSTALL_LABEL = "   "
     INSTALL_LABEL2 = "   "
     STARTING_LABEL = "   "
 
 # Disable colors, for the color blind and others who do not prefer colors.
-if environments.PIPENV_COLORBLIND:
-    crayons.disable()
+#if environments.PIPENV_COLORBLIND:
+
```

Comment:
this is the spot where it previously switched the NO_COLOR behavior

Reply:
![image](https://user-images.githubusercontent.com/1083045/180061887-d0df1d6c-0c97-4b1b-87b0-cae96c4fd9ab.png)

Reply:
I am implementing a conditional monkeypatch in this spot on style and secho, they are used separately for composing messages that aren't all one color, also tests for each for the old and new variables.  Should an issue be created for deprecating PIPENV_COLORBLIND?